packages/cli/src/telemetry/index.ts

```diff
@@ -111,18 +111,22 @@ async function checkOptIn(interactive: boolean): Promise<GlobalTelemetryOptions
   return globalOptions;
 }
 
-function getUserEnvironment(): UserEnvironment {
+async function getUserEnvironment(): Promise<UserEnvironment> {
   return {
     platform: process.platform,
     arch: process.arch,
     nodeVersion: process.version,
-    cliVersion: require('../../package.json').version,
+    cliVersion: await getCLIVersion(),
     upgradesVersion: getDependencyVersion('@openzeppelin/upgrades'),
     truffleVersion: getDependencyVersion('truffle'),
     web3Version: getDependencyVersion('web3'),
   };
 }
 
+async function getCLIVersion(): Promise<string> {
+  return JSON.parse(await fs.readFile(__dirname + '/../../package.json', 'utf8')).version;
```

Comment:
Why not [readJson](https://github.com/jprichardson/node-fs-extra/blob/HEAD/docs/readJson.md) from fs-extra?

Reply:
It got past me!